apps/web-giddh/src/app/invoice/settings/invoice.settings.component.ts

```diff
@@ -86,6 +86,8 @@ export class InvoiceSettingComponent implements OnInit, OnDestroy {
     public selectedTabIndex: number = 0;
     /** Active tab name */
     public activeTab: string;
+    /** Active company details */
+    public selectedCompany: any = null;
```

Comment:
```suggestion
    public activeCompany: any = null;
```

Please make changes in overall file

Reply:
Done